packages/fast-components-styles-msft/src/utilities/design-system.ts

```diff
@@ -9,6 +9,20 @@ export const backgroundColor: DesignSystemResolver<string> = getDesignSystemValu
     "backgroundColor"
 );
 
+/**
+ * Retrieve the base neutral color from the design system
```

Comment:
you're putting "." at the end of your other block comments, is there a reason you're doing so some of the time but not here?

Reply:
I was cleaning up the other file while I was in it. I'll save this file for another time. ;)